public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -28,6 +30,9 @@ interface State {
   // Data frem the last response
   data: DataFrame[];
 
+  // Error from query
+  error: DataQueryError;
+
```

Comment:
do we need a special entry for error here?  or just check `last.error`?  Eventually the `last` variable should be better typed....

Reply:
Agree with @ryantxu, we probably don't need this.